csaf/src/report/render.rs

```diff
@@ -35,7 +38,7 @@ impl Display for Title {
     }
 }
 
-struct HtmlReport<'r>(&'r ReportResult<'r>, &'r RenderOptions);
+struct HtmlReport<'r>(&'r ReportResult<'r>, &'r PathBuf, &'r Option<Url>);
```

Comment:
As we are adding more fields, I think it makes sense to use a struct variant `{}` instead of tuple.

Reply:
like this csaf-walker::report::render,
https://github.com/trustification/trustification/pull/1064/files#diff-ef9fdbc1979542d4bd996a2209d42c407d7e0255ff2340390c6179a3179ee154R8